starport/pkg/spn/spn.go

```diff
@@ -347,15 +371,31 @@ func (c *Client) ChainGet(ctx context.Context, accountName, chainID string) (Cha
 		genesisAccounts = append(genesisAccounts, genesisAccount)
 	}
 
-	return Chain{
-		URL:             res.Chain.SourceURL,
-		Hash:            res.Chain.SourceHash,
+	return LaunchInformation{
 		Peers:           launchInformationRes.Peers,
 		GenesisAccounts: genesisAccounts,
 		GenTxs:          launchInformationRes.GenTxs,
 	}, nil
 }
 
+
+// GetChain gets the overall data of a chain
+func (c *Client) GetChain(ctx context.Context, accountName, chainID string) (Chain, error) {
```

Comment:
These new namings and funcs in `spn.go` confuses me a bit.

`GetChain` used to just call `ShowChain` on SPN to get [Chain](https://github.com/tendermint/spn/blob/1e20634/proto/genesis/v1beta/types.proto#L9) type.

Right now I see that `GetChain` renamed to `GetChainInformation` and new `GetChain` calls `GetChainInformation` and `GetLaunchInformation` to create a `Chain`. Also, `Chain` type's structure is now different comparing to [one](https://github.com/tendermint/spn/blob/1e20634/proto/genesis/v1beta/types.proto#L9) in SPN. 

For the sake of simplicity, I rather to keep the old `Chain` type and old `GetChain` but also update it to with latest changes in SPN.
Then we can have a new `GetLaunchInformation` that calls corresponding [rpc](https://github.com/tendermint/spn/blob/develop/proto/genesis/v1beta/querier.proto#L17) in SPN and returns the same type there.

Reply:
`GetChain` is actually not used here.

Before we called `ShowChain` and `LaunchInformation` from `spn` in the same function while the data from the function was used either for the chain static info (source URL or source Hash) or the launch information (gentxs, accounts, peers). So the function has been splitted so that we call only the necessary query.

I only put `sourceURL` and `sourceHash` in `ChainInformation` because it was the only piece of information used so far. But if we want to have the exact structure as on `spn` why wouldn't we import directly the type from `spn` package.